tests/cli/workflow/mip/test_cli_mip_base.py

```diff
@@ -0,0 +1,118 @@
+""" Test the CLI for run mip-dna """
+
+import logging
+
+from cg.meta.workflow.mip import MipAnalysisAPI
+from cg.meta.workflow.prepare_fastq import PrepareFastqAPI
+from cg.cli.workflow.mip_dna.base import start, start_available
+from cg.store.api.status import StatusHandler
+
+from tests.meta.conftest import fixture_analysis_api
+from tests.meta.conftest import fixture_mip_hk_store
+
+
+def test_spring_decompression_needed_and_started(
+    mocker, cli_runner, caplog, dna_mip_context, case_id, analysis_api
+):
+    caplog.set_level(logging.INFO)
+
```

Comment:
Please add GIVEN WHEN THEN clauses to tests explaining the setups and what mockers do and why

Reply:
Yes every time you modify below write `#GIVEN a case that exists in statusdb` etc

Reply:
You could also add a docstring to the test that briefly describes what os being tested